packages/hoppscotch-app/locales/en.json

```diff
@@ -44,7 +44,10 @@
     "chat_with_us": "Chat with us",
     "contact_us": "Contact us",
     "copy": "Copy",
+    "copy_user_id": "Copy user id",
```

Comment:
Something better like, "Copy User Auth Token". This is not an ID, its a token foremost.